frontend/src/components/TicketCardClose.tsx

```diff
@@ -0,0 +1,239 @@
+  width: '12px',
+  height: '12px',
+  borderRadius: '50%',
+  border: '2px solid #fff',
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+});
+
+const InnerCircle = styled(Box)({
+  width: '6px',
+  height: '6px',
+  borderRadius: '50%',
+  backgroundColor: '#fff',
+});
+
+const LocationInfo = styled(Box)({
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  paddingBottom: '8px',
+});
+
+const Location = styled(Box)({
+  display: 'flex',
+  flexDirection: 'column',
+});
+
+const DoneStamp = styled(Box)({
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%) rotate(-20deg)',
+  color: '#fff',
+  fontSize: '2rem',
+  fontWeight: 'bold',
+  opacity: 0.2,
+  textTransform: 'uppercase',
+  textAlign: 'center',
+  zIndex: 10,
+});
+
+const FadedCircle = styled(Box)({
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: '200px',
+  height: '200px',
+  borderRadius: '50%',
+  border: '4px dashed #fff',
+  opacity: 0.2,
+  zIndex: 9,
+});
+
+const BoldText = styled(Typography)({
+  fontWeight: '600',
+  fontSize: '1.125rem',
+  font: 'Outfit',
+  color: '#FFFFFF',
+});
+const BoldTextBottom = styled(Typography)({
+  fontWeight: '500',
+  fontSize: '9px',
+  font: 'Outfit',
+  WebkitTextStrokeWidth: '0.5px',
+  lineHeight: '11.34px',
+  color: ' #FFFFFF',
+});
+
+const Time = styled(Typography)({
+  fontSize: '1.5rem',
+  fontWeight: 'bold',
+  color: '#FFFFFF',
+});
+
+const BoldFooterText = styled(Typography)({
+  fontWeight: '500',
+  fontSize: '0.5625rem',
+});
+
+interface TicketCardCloseProps {
+  busNumber: string;
+  departure: string;
+  destination: string;
+  time: string;
+  date: string;
+  ticketId: string;
+}
+
+const TicketCardClose: React.FC<TicketCardCloseProps> = ({
+  busNumber,
+  departure,
+  destination,
+  time,
+  date,
+  ticketId,
+}) => {
+  return (
+    <Root>
+      <Header>
+        <HeaderText>{ticketId}</HeaderText>
+      </Header>
+      <FadedCircle />
+      <DoneStamp>DONE</DoneStamp>
+      <Content>
+        <LocationInfo>
+          <Location>
+            <BoldText>{departure}</BoldText>
+            <BoldTextBottom>PDPM IIITDM Jabalpur</BoldTextBottom>
+          </Location>
+          <Location style={{textAlign: 'right'}}>
+            <BoldText align="right">{destination}</BoldText>
+            <BoldTextBottom align="right">Jabalpur City</BoldTextBottom>
+          </Location>
+        </LocationInfo>
+        <DashedLine>
+          <Circle>
+            <InnerCircle />
+          </Circle>
+          <DashedSegment />
+          <DashedSegment />
+          <DashedSegment />
+          <DashedSegment />
+          <DashedSegment />
+          <DashedSegment />
+          <DashedSegment />
+          <DashedSegment />
+          <DashedSegment />
+          <DashedSegment />
+          <DashedSegment />
+          <DashedSegment />
```

Comment:
Since we are keeping the overflow visible, we need to conditionally render the number of DashedSegment on the basis of screen size to make the card responsive for mobile devices. You can create an array which length will be decided by the screensize( keep 5 for mobiledevices other 11) then render the component accordingly